example/example.cpp

```diff
@@ -22,32 +24,48 @@ capo::Vec3 UCMVelocity(float angularSpeed, float time, float radius) {
 	return {-radius * angularSpeed * std::cos(time * angularSpeed), radius * angularSpeed * std::sin(time * angularSpeed), 0};
 }
 
-int openAlTest(std::string const& wavPath, float gain, bool loop) {
+bool openAlTest(std::string const& wavPath, float gain, bool loop) {
 	capo::Instance instance;
 	if (!instance.valid()) {
 		std::cerr << "Couldn't create valid instance." << std::endl;
-		return impl::fail_code;
+		return false;
 	}
 
 	auto pcm = capo::PCM::fromFile(wavPath);
 	if (!pcm) {
-		std::cerr << "Couldn't load audio file." << std::endl;
-		return impl::fail_code;
+		switch (pcm.error()) {
+		case capo::Error::eUnknownFormat:
+			static_assert(static_cast<std::size_t>(capo::FileFormat::eCOUNT_) == 4, "Unhandled file format");
```

Comment:
This static assert seems unnecessary...

Reply:
It's tied to the next output message. If we end up supporting more formats we'll need to add them manually to the supported formats cout